### PR TITLE
Feature/automation 13

### PR DESCRIPTION
**Pull request by JMCullimoreDell**
Dockerfile and config (for regex rewriting players to be served from here rather than the engine) for building runtime image from nginx

### CODE REVIEW

File: conf/default.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    location /players/ {
+        root /usr/share/nginx/html;
+        location ~* {
+            add_header Content-Type text/html;
+            default_type text/html;
+        }
+    }
+
+    rewrite ^/(\w\w\w\w\w\w\w\w-\w\w\w\w-\w\w\w\w-\w\w\w\w-\w\w\w\w\w\w\w\w\w\w\w\w)/play/(.*) /players/$1.$2 last;
```
Comment (itmecho) [inline conf/default.conf:18]: This regex could be rewritten to 
```
rewrite ^/(.+)/play/(.+) /players/$1.$2 last;
```

Comment (JMCullimoreDell) [reply conf/default.conf:18]: yes I'd have preferred to have written:
\w{8}-\w{4}-\w{4}-\w{4}-\w{12}
but nginx config breaks when you put { or } in there, thought I'd keep it more inline with guids as that was the frame of mind I was in when writing it and it does work and doesn't use greedy expansion so is probably a little bit quicker

**Review by dshoulders — APPROVED** [review]
Will not have any effect on local or production running code.

File: conf/default.conf (hunk 1 above)
Comment (jonjomckay) [reply conf/default.conf:18]: I agree with @itmecho here - the regex should be a lot simpler than what's there. It doesn't matter to nginx whether it's a UUID or not - it just needs the match groups. @JMCullimoreDell can you make the change in another branch and PR please?

**Pull request merged**